README.md

```diff
@@ -415,6 +416,27 @@ timeAgo('1400/04/07 18:00:00') // حدود 3 هفته  بعد
 timeAgo('1401/03/17 18:00:00') // حدود 1 سال  بعد
 ```
 
+### Get the Remaining Time of the Date 
+
+**Usage**
+
+>Takes a date(it could be string, number or date) and calculate years,
+>months, days, hours, minutes and seconds remained to that specific date. 
+
+```js 
+import { remainingTime } from "@persian-tools/persian-tools";
+
+// CurrentDateTime: () => new Date("2022-04-12T10:30:51Z")
+
+remainingTime("2023-05-14T13:35:59Z").toString() // ۱ سال و ۱ ماه و ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
+remainingTime("2022-05-14T13:35:59Z").toString() // ۱ ماه و ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
+remainingTime("2022-04-14T13:35:59Z").toString() // ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
```

Comment:
Please write other methods and results such as `RemainingTime` object and `IsFinished`

Reply:
And there is no need to write more than 4 samples.

Reply:
check the changes